packages/pagination/src/usePagination.ts

```diff
@@ -43,10 +43,8 @@ export function usePagination<Item = any>(
     getItemProps
   } = useSelection(options);
 
-  const getContainerProps = ({ role = 'navigation', ariaLabel, ...props } = {} as any) => {
+  const getContainerProps = ({ ...props } = {} as any) => {
     return {
-      role,
-      'aria-label': ariaLabel || 'Pagination navigation',
```

Comment:
I removed the default value, the string "Pagination navigation" because I don't think it's necessary. Consumers still have the option to pass a value into the prop getter for `aria-label` if they choose to.

Reply:
With a wrapping `<nav>` this would already read off as a "Navigation area" so it probably wasn't needed. 👍